ts/features/itwallet/machine/credential/guards.ts

```diff
@@ -21,5 +26,19 @@ export const createCredentialIssuanceGuardsImplementation = () => ({
     ),
 
   isStatusError: ({ context }: { context: Context }) =>
-    context.failure?.type === CredentialIssuanceFailureType.INVALID_STATUS
+    context.failure?.type === CredentialIssuanceFailureType.INVALID_STATUS,
+
+  isSkipNavigation: ({ event }: { event: CredentialIssuanceEvents }) =>
+    event.type === "select-credential" && event.skipNavigation === true,
+
+  isEidExpired: () => {
+    const eid = itwCredentialsEidSelector(store.getState());
+
+    return pipe(
+      eid,
+      O.map(getCredentialStatus),
+      O.map(status => status === "jwtExpired"),
+      O.getOrElse(() => false)
+    );
```

Comment:
We have a selector to get directly the eID status

```suggestion
    const eidStatus = itwCredentialsEidStatusSelector(store.getState());
    return eidStatus === "jwtExpired"
```